pkg/application/registryConfigUseCase.go

```diff
@@ -0,0 +1,300 @@
+package application
+
+import (
+	"context"
+	_ "embed"
+	"errors"
+	"fmt"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/common"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/ecosystem"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domainservice"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+//go:embed testdata/doguConfigDiffMockData.yaml
+var doguConfigDiffMockDataBytes []byte
+
+//go:embed testdata/globalConfigDiffMockData.yaml
+var globalConfigDiffMockDataBytes []byte
+
+var sensitiveDoguConfigEntryError = fmt.Errorf("sensitive dogu config error")
```

Comment:
🚫 **[checkstyle]** <errname> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>the variable name `sensitiveDoguConfigEntryError` should conform to the `errXxx` format